modules/ossm-control-plane-verify.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/installing-ossm.adoc
+
+[id="ossm-control-plane-verify_{context}"]
+
+= Verifying the control plane installation
+
+Verify that the control plane installation completed successfully.
+
+.Procedure
+
+[NOTE]
+====
+The name of the resource is `istio-installation`.
+====
+
+. Run this command to determine if the Operator finished deploying the control plane:
++
+----
+$ oc get servicemeshcontrolplane/basic-install -n istio-system --template='{{range .status.conditions}}{{printf "%s=%s, reason=%s, message=%s\n\n" .type .status .reason .message}}{{end}}'
+----
++
+When the control plane installation is finished, the output is similar to the following:
++
+----
+Installed=True, reason=InstallSuccessful, message=%!s()
+
+Reconciled=True, reason=InstallSuccessful, message=%!s()
+----
+
+. After the control plane is deployed, run this command to check the status of the pods:
++
+----
+$ oc get pods -n istio-system
+----
+
+. Verify that the pods are in a state similar to this:
++
+----
+NAME                                          READY     STATUS      RESTARTS   AGE
+3scale-istio-adapter-7df4db48cf-sc98s         1/1       Running     0          13s
```

Comment:
This should now look like: 

NAME                                      READY   STATUS             RESTARTS   AGE
3scale-istio-adapter-67b96f97b5-cwvgt     1/1     Running   0          99s
grafana-75f4cbbc6-xw99s                   2/2     Running            0          48m
istio-citadel-8489b8bb96-ttqfd            1/1     Running            0          54m
istio-egressgateway-5ccd4d5ddd-wtp2h      1/1     Running            0          52m
istio-galley-58ff8db57c-jrpkz             1/1     Running            0          54m
istio-ingressgateway-698674848f-bk57s     1/1     Running            0          52m
istio-node-2d764                          1/1     Running            0          54m
istio-node-4h926                          1/1     Running            0          54m
istio-node-6qxcj                          1/1     Running            0          54m
istio-node-8fxqz                          1/1     Running            0          54m
istio-node-gzg5v                          1/1     Running            0          54m
istio-node-vxx5p                          1/1     Running            0          54m
istio-pilot-764966cf69-9nlhp              2/2     Running            0          19m
istio-policy-7c856f7d5f-4fjk4             2/2     Running            0          53m
istio-sidecar-injector-757b8ccdbf-znggc   1/1     Running            0          49m
istio-telemetry-65d8b47c98-jrp9h          2/2     Running            0          53m
jaeger-f775b79f8-cmbb2                    2/2     Running            0          54m
kiali-7646d796cd-kfx29                    1/1     Running            0          20m
prometheus-56cb9c859b-dlqmn               2/2     Running            0          54m

Reply:
If we display this amount of "istio-node-*" it is important to note that this depends on the number of nodes in the cluster, right? Otherwise operators with less or more nodes may compare and be confused by the amount of nodes. It may be obvious for some, but not for others.

Reply:
yeah, a lot of this varies depending on configuration. 3scale, jaeger, kiali, and prometheus may or not be present, the number of istio-nodes instances changes with the number of systems in the cluster, and the Istio components can be turned on and off individually as well. Don't know if "verify that the pods are in a similar state to..." is enough or not.

Reply:
I updated the table and I will make sure to add a note that explains that the verification results vary depending on the configuration.